### PR TITLE
changing images and some cleanup

### PR DESCRIPTION
**Pull request by sumitarora**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,20 +38,20 @@ you should read it.
...
+README just for developers](https://github.com/rangle/augury/wiki). Jumping into a new codebase
 can be daunting so we've tried, as best as we could, to give you an
-overview into the design of Batarangle and our intentions. It's a
+overview into the design of Augury and our intentions. It's a
 glimpse into our heads, if you will. We're not yet sure that's a good
 thing but we can assure you it's a useful one.
 
 ### The chat room
 
 :speech_balloon: [Slack](http://batarangle-slack.herokuapp.com/)
```
Comment (winkerVSbecks) [inline CONTRIBUTING.md]: does the slack room need to be re-named?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ wait to hear from you.
 
 :bangbang: [Code of Conduct](./CODE_OF_CONDUCT.md)
 
-In the interest of making the Batarangle project a safe and friendly
+In the interest of making the Augury project a safe and friendly
 place for people from diverse backgrounds, we'll be adhering to a [Contributor Code of Conduct](./CODE_OF_CONDUCT). As a contributor, you'll be expected to
 uphold this code as well as report unacceptable behaviour to
 [batarangle@rangle.io](mailto:batarangle@rangle.io).
```
Comment (winkerVSbecks) [inline CONTRIBUTING.md]: same for the email address?

File: CONTRIBUTING.md (hunk 1 above)
Comment (sumitarora) [inline CONTRIBUTING.md]: yes @bertrandk needs to change heroku app as well as domain etc. working on those still

**Comment by winkerVSbecks** [comment]
:shipit:

**Pull request merged**